tests/lib/structures/default-provider/MapProvider.test.ts

```diff
@@ -1707,17 +1707,20 @@ describe('MapProvider', () => {
       });
 
       test('GIVEN provider w/ data THEN loops x times', async () => {
-        const mockCallback = jest.fn(() => true);
+        const mockCallback = jest.fn((..._) => true);
```

Comment:
What is this for? `..._`

Reply:
@RealShadowNova It's so that the arguments are checked later on; otherwise TS will complain of the call being messed up.